rust/gui-client/README.md

```diff
@@ -75,6 +75,23 @@ If that doesn't work:
 - Click `New client secret`
 - Note down the secret value. This should be entered into the GitHub repository's secrets as `AZURE_CLIENT_SECRET`.
 
+## Installing on CentOS 9
+
+These instructions will move to the knowledge base once the first release supporting CentOS 9 is cut.
+
+1. Download RPM
+2. `sudo dnf install systemd-resolved` (Installing it explicitly prevents it from being auto-removed if Firezone is removed)
```

Comment:
I think this can be interesting to point out but seems like a decision the user should make? On Debian-based distros, we just auto-install it, no?

Reply:
On Ubuntu we assume it to already be installed. I got a shock on CentOS when I un-installed Firezone and `/etc/resolv.conf` was still pointing to systemd-resolved, but it had apparently un-installed systemd-resolved too. (I think apt would do the same in this situation)

Reply:
Yeah probably. I'd expect the systemd-resolved uninstaller to fix the symlink :(